src/style/palette.rs

```diff
@@ -1,4 +1,11 @@
-use super::color::PaletteColor;
+use super::{color::PaletteColor, full_palette};
```

Comment:
Note that full_palette is optional, so a conditional complication is required at this point